docs/languages/cpp.md

```diff
@@ -194,57 +143,77 @@ To go to a symbol's definition, place your cursor on the symbol anywhere its use
 
 ## Debugging
 
+After you have set up the basics of your debugging environment as specified in [Getting Started](#getting_started), you can learn more details about debugging C/C++ here.
+
 VS Code supports the following debuggers for C/C++ depending on the operating system you are using:
 
-* **Linux**: supports debugging using GDB
-* **OS X**: supports using LLDB or GDB
-* **Windows**: currently supports GDB only (using Cygwin or MinGW)
+* **Linux**: using GDB
+* **OS X**: using LLDB or GDB
+* **Windows**: using the Visual Studio Windows Debugger or GDB (using Cygwin or MinGW)
 
-### Windows debugging (Cygwin/MinGW) <a name="debug_windows"></a>
+### Windows Debugging on Cygwin/MinGW <a name="debug_windows"></a>

```

Comment:
Renamed the anchors and title.

No section for VSWD because there isn't anything special to mention about it.